src/functionaltest/java/com/ericsson/ei/frontend/TestRulesFunctionality.java

```diff
@@ -30,102 +30,116 @@
     private static ClientAndServer mockServer;
     private static final String BASE_URL = "localhost";
 
-    private static final String DOWNLOADED_RULES_TEMPLATE_FILE_PATH = String.join(
-            File.separator, SeleniumConfig.getTempDownloadDirectory().getPath(), "rulesTemplate.json");
-    private static final String RULES_TEMPLATE_FILE_PATH = String.join(
-            File.separator, "src", "functionaltest", "resources", "responses", "RulesTemplateObject.json");
-    private static final String DOWNLOADED_RULES_FILE_PATH = String.join(
-            File.separator, SeleniumConfig.getTempDownloadDirectory().getPath(), "rules.json");
-    private static final String DOWNLOADED_EVENTS_TEMPLATE_FILE_PATH = String.join(
-            File.separator, SeleniumConfig.getTempDownloadDirectory().getPath(), "eventsTemplate.json");
-    private static final String EVENTS_TEMPLATE_FILE_PATH = String.join(
-            File.separator, "src", "functionaltest", "resources", "responses", "EventsTemplateObject.json");
-    private static final String AGGREGATED_OBJECT_FILE_PATH = String.join(
-            File.separator, "src", "functionaltest", "resources", "responses", "AggregatedObjectResponse.json");
+    private static final String DOWNLOADED_RULES_TEMPLATE_FILE_PATH = String.join(File.separator, SeleniumConfig.getTempDownloadDirectory().getPath(),
+            "rulesTemplate.json");
+    private static final String RULES_TEMPLATE_FILE_PATH = String.join(File.separator, "src", "functionaltest", "resources", "responses",
+            "RulesTemplateObject.json");
+    private static final String DOWNLOADED_RULES_FILE_PATH = String.join(File.separator, SeleniumConfig.getTempDownloadDirectory().getPath(),
+            "rules.json");
+    private static final String DOWNLOADED_EVENTS_TEMPLATE_FILE_PATH = String.join(File.separator,
+            SeleniumConfig.getTempDownloadDirectory().getPath(), "eventsTemplate.json");
+    private static final String EVENTS_TEMPLATE_FILE_PATH = String.join(File.separator, "src", "functionaltest", "resources", "responses",
+            "EventsTemplateObject.json");
+    private static final String AGGREGATED_OBJECT_FILE_PATH = String.join(File.separator, "src", "functionaltest", "resources", "responses",
+            "AggregatedObjectResponse.json");
 
     @MockBean
     protected CloseableHttpClient mockedHttpClient;
 
-    @Test
-    public void testJourneyToFindAggregatedObjectButton() throws Exception {
-        // Set up
-        int portServer = mockServer.getLocalPort();
+    private JavascriptExecutor js;
```

Comment:
Please variable name.

Reply:
Removed it since the driver could execute scripts as well.